docs-site/docs/configuration/ga4gh_wes.md

```diff
@@ -4,34 +4,72 @@ id: ga4gh_wes
 title: Connecting to a GA4GH WES server
 ---
 
-### Setup
+### Server Connection
 
-The following fields can be set in the rails credentials file.
+To connect IRIDA Next to a GA4GH Server, you will need to edit your rails credentials file. This file is encrypted so secret tokens are secure.
+
+Edit your production credentials file with the following command.
+
+`EDITOR="vim --nofork" bin/rails credentials:edit --environment production`
+
+[Read more about the rails credential file here](https://guides.rubyonrails.org/security.html#custom-credentials)
+
+The following fields can be set.
 
 ```yml
 ga4gh_wes:
+  server_url_endpoint: 'https://<some server url>/wes/1.01/'
   oauth_token: <some oauth token>
   headers: { '<some header key>': '<some header value>' }
-  server_url_endpoint: 'https://<some server url>/wes/1.01/'
 ```
 
-#### oauth_token
+Only `server_url_endpoint` is required by default, but depending on your GA4GH WES server configuration the others might also be needed.
```

Comment:
Can we change this to:

Only `server_url_endpoint` is required by default, but depending on your GA4GH WES server configuration the other fields may also be required.

Reply:
Fixed in https://github.com/phac-nml/irida-next/pull/628/commits/5692b2afe9fb6da61e753dfe1d2d26dec66c4221